Solutions/Vectra AI Stream/Data Connectors/template_VectraStreamAma.json

```diff
@@ -0,0 +1,246 @@
+      "query": "vectra_dns_CL \n| where rcode_name == \"NXDomain\"\n| summarize Count=count() by query\n| order by Count desc\n| limit 10"
+    },
+    {
+      "description": "Host and Web sites using non-ephemeral Diffie-Hellman key exchange",
+      "query": "vectra_ssl_CL \n| where cipher contains \"TLS_RSA\"\n| distinct orig_hostname, id_orig_h, id_resp_h, server_name, cipher\n| project orig_hostname, id_orig_h, id_resp_h, server_name, cipher"
+    }
+  ],
+  "dataTypes": [
+    {
+      "name": "vectra_isession",
+      "lastDataReceivedQuery": "vectra_isession\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_dns",
+      "lastDataReceivedQuery": "vectra_dns\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_ssl",
+      "lastDataReceivedQuery": "vectra_ssl\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_beacon",
+      "lastDataReceivedQuery": "vectra_beacon\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_dcerpc",
+      "lastDataReceivedQuery": "vectra_dcerpc\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_dhcp",
+      "lastDataReceivedQuery": "vectra_dhcp\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_http",
+      "lastDataReceivedQuery": "vectra_http\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_kerberos",
+      "lastDataReceivedQuery": "vectra_kerberos\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_ldap",
+      "lastDataReceivedQuery": "vectra_ldap\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_ntlm",
+      "lastDataReceivedQuery": "vectra_ntlm\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_radius",
+      "lastDataReceivedQuery": "vectra_radius\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_rdp",
+      "lastDataReceivedQuery": "vectra_rdp\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_smbfiles",
+      "lastDataReceivedQuery": "vectra_smbfiles\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_smbmaping",
+      "lastDataReceivedQuery": "vectra_smbmaping\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_smtp",
+      "lastDataReceivedQuery": "vectra_smtp\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_ssh",
+      "lastDataReceivedQuery": "vectra_ssh\n |  summarize Time = max(TimeGenerated)"
+    },
+    {
+      "name": "vectra_x509",
+      "lastDataReceivedQuery": "vectra_x509\n |  summarize Time = max(TimeGenerated)"
+    }
+  ],
+  "connectivityCriterias": [
+    {
+      "type": "IsConnectedQuery",
+      "value": [
+        "\nvectra_stream\n| summarize LastLogReceived = max(TimeGenerated)\n| project IsConnected = LastLogReceived > ago(1h)"
```

Comment:
does the table should be VectraStream in Connectivity Criteria method 

![image](https://github.com/Azure/Azure-Sentinel/assets/117061676/01da86bd-f74f-4d8a-a821-c206417d6e4c)

Just like other connector ?

Reply:
In the update to AMA connector, we decided to split the different network metadata types to multiple tables (instead of previously one named VectraStream_CL). Now, the function to look across all tables (metadata types) is vectra_stream. See this file Solutions/Vectra AI Stream/Parsers/vectra_stream.yaml.

Reply:
Then @danymello missed to add a custom table named vectra_stream 
please add the table in custom table folder. 
2 tables with name vectra_stream && vectra_stream_CL

Reply:
@v-prasadboke, it is not a custom table, it is a KQL with union of all other tables: https://github.com/Azure/Azure-Sentinel/blob/a663fef0b6cdeec1362dad55c44464f3a8756465/Solutions/Vectra%20AI%20Stream/Parsers/vectra_stream.yaml 

As there is no table in Analytics workspace named vectra_stream_CL. What do you recommend to do? Should I based the connectivity test on an actual table? for example vectra_isession_CL

Reply:
I'll try to resolve it from my end @fgu-vectra